src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+	"event_message_args_missing_list": "{emote_warn} Fehlende argumente `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Richtige Benutzung:** {usage}",
+	"event_message_edit": "{emote_edit} Nachricht von **{target_tag}** `{target_id}` wurde Bearbeitet in <#{channel_id}>\n`ID (channel-message): {after_channel_id}-{after_id}`\n**B:** {before}\n**A:** {after}",
+	"event_message_delete": "{emote_trash} Nachricht von **{target_tag}** `{target_id}` wurde gelöscht in <#{channel_id}>\n`ID (channel-message): {after_channel_id}-${after_id}`\n**C:** {content} \n**A:** {attachment}\n**E:** {embed}",
+
+	"event_channel_create": "**Kanal erstellt:** <#{channel_id}> `({channel_type})`",
+	"event_channel_delete": "**Kanal geköscht:** #{channel_name} `({channel_id})` `({channel_type})`",
+	"event_channel_update_topic_added": "<#{channel_id}> Kanal Thema wurde hinzugefügt `{channel_topic}`\n",
+	"event_channel_update_topic_removed": "<#{channel_id}> Kanal Thema wurde entfernt\n",
+	"event_channel_update_topic_updated": "<#{channel_id}> Kanal Thema wurde von `{oldchannel_topic}` zu `{newchannel_topic} geändert`\n",
+	"event_channel_update_name": "<#{channel_id}> Name wurde von `#{oldchannel_name}` zu `#{newchannel_name} geändert`\n",
+	"event_channel_update_type": "<#{channel_id}> Art wurde von `{oldchannel_type}` zu `{newchannel_type} geändert`\n",
+	"event_channel_update_nsfw_enabled": "<#{channel_id}> nsfw wurde `aktiviert`\n",
+  "event_channel_update_nsfw_disabled": "<#{channel_id}> nsfw wurde `deaktiviert`\n",
+
+	"event_member_joined": "{emote_join} **Neues Mitglied:** **{user_tag}** `({user_id})` - **Account erstellt:** {user_age}",
+	"event_member_left": "{emote_leave} **Mitglied hat uns Verlassen:** **{user_tag}** `({user_id})`",
+
+	"event_member_update_nickname": "{emote_warn} **{user_tag}** `({user_id})` hat seinen Nicknamen von `{nick_old}` zu `{nick_new} geändert`",
+	"event_member_update_role_add": "{emote_warn} Die Rolle **{role}** wurde zu  **{user_tag}**  `({user_id}) hinzugefügt`",
+	"event_member_update_role_remove": "{emote_warn} Die Rolle **{role}** wurde von **{user_tag}** `({user_id}) entfernt`",
+	"event_member_update_role_add_audit": "{emote_warn} Die Rolle **{role}** Wurde zu **{user_tag}** `({user_id}) hinzugefügt` von **{moderator_tag}** `({moderator_id})`",
+	"event_member_update_role_remove_audit": "{emote_warn} Die Rolle **{role}** wurde entfernt von **{user_tag}** `({user_id})` von **{moderator_tag}** `({moderator_id})`",
+
+	"config_language_success": "{emote_success} Sprache wurde erfolgreicht zu: `{language} geändert`",
+	"config_language_invalid_args": "{emote_warn} Falsche Sprache angegeben!\n{emote_wrench} Wähle eine Sprache von `{languages}`",
+
+	"config_logging_success": "{emote_success} Erfolgreich Loggin {part} in {channel} aktiviert",
+	"config_logging_invalid_channel": "{emote_warn}  `{channel_name}` ist kein gültiger Kanal!",
```

Double Space, and translation, not explanation.
```suggestion
	"config_logging_invalid_channel": "{emote_warn} `{channel_name}` existiert nicht!",
```